samples/snippets/src/main/java/com/google/cloud/translate/automl/DatasetApi.java

```diff
@@ -43,156 +39,6 @@
  */
 public class DatasetApi {
 
-  // [START automl_translate_create_dataset]
```

Comment:
We need to keep at least one copy of automl_translate_create_dataset around.

Reply:
https://github.com/googleapis/java-automl/blob/fb9dad14aac1d684ea136ccca158798545ea26f3/samples/snippets/src/main/java/beta/automl/TablesCreateDataset.java#L19